course/src/main/java/org/openedx/course/presentation/container/CourseContainerViewModel.kt

```diff
@@ -248,7 +243,7 @@ class CourseContainerViewModel(
     fun updateData() {
         viewModelScope.launch {
             try {
-                interactor.preloadCourseStructure(courseId)
+                interactor.getCourseStructure(courseId)
```

Comment:
Will it actually update the data, or will it just return the last cached course?

Reply:
no, but I changed how it works and right now it should update the data )